tool/matomo/classes/tool/tool.php

```diff
@@ -41,6 +41,8 @@ public function get_tracking_code() {
         $template = new \stdClass();
         $template->siteid = $settings['siteid'];
         $template->siteurl = $settings['siteurl'];
+        $usepiwik = (isset($settings['piwikjsurl']) && !empty($settings['piwikjsurl']));
```

Comment:
Sorry, John.  But $usepiwik is confusing as well. I should point to it first time. Maybe $custompiwikjs?